src/lilURL.php

```diff
@@ -61,13 +69,46 @@ public function handleRedirect($id)
         if ($id != '' && $id != basename($_SERVER['PHP_SELF']) && $id != '?login') {
             $location = $this->getURL($id);
             if ($location != false) {
+                $this->trackHit($id);
```

Comment:
I'm worried that placing blocking code that depends on a network request before redirecting will cause problems at some point down the road (slow redirects or even failing redirects).

Have you considered running the tracking hit with `->setAsyncRequest(true)` or queuing tracking events to happen somewhere outside of the request workflow (cron job)?